public/scripts/helper.js

```diff
@@ -0,0 +1,25 @@
+// https://stackoverflow.com/questions/14573223/set-cookie-and-get-cookie-with-javascript
+export function setCookie(name,value,days) {
+    var expires = "";
+    if (days) {
+        var date = new Date();
+        date.setTime(date.getTime() + (days*24*60*60*1000));
+        expires = "; expires=" + date.toUTCString();
+    }
+    document.cookie = name + "=" + (value || "")  + expires + "; path=/";
+}
+
+export function getCookie(name) {
+    var nameEQ = name + "=";
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
+        var c = ca[i];
```

Comment:
I would prefer the usage of let as often as possible over var, as var's are outdated
```suggestion
        let c = ca[i];
```